src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/licences/AuditEventRepository.kt

```diff
@@ -13,4 +13,10 @@ interface AuditEventRepository : JpaRepository<AuditEvent, Long>, JpaSpecificati
     value = "select id, timestamp, user, action, details from audit where details ->> 'bookingId' in :bookingIds order by timestamp desc;",
   )
   fun findByBookingIds(bookingIds: Collection<String>): List<AuditEvent>
+
+  @Query(
+    nativeQuery = true,
+    value = "insert into audit user, action, details;",
+  )
```

Comment:
Not sure if I also insert a `timestamp` in this query or if this gets added and populated automatically like I think `id` does? 😅